poster/src/index.ts

```diff
@@ -49,10 +36,10 @@ async function run() {
   const price_deltas = JSON.parse(parsed['price-deltas']);
   const assets = <string[]>parsed['asset'];
 
-  // check that price deltas are set up for all assets, otherwise set default delta value
+  // check that price deltas are set up for all assets
   assets.forEach(asset => {
     if (price_deltas[asset] == undefined) {
-      price_deltas[asset] = defaultDeltas[asset];
+      throw new TypeError(`For each asset price delta should be provided, ${asset} asset is not properly configured`)
```

Comment:
Only question is: maybe we just want to have a sane default to make life easy, like 5%? Not that we're using it, but it's easier than getting yelled at. Thoughts?

Reply:
that's what we had before, but then I switched it to different deltas. I would say let's just set all deltas in the poster, it's clear approach and it forces us to properly configure poster and really understand deltas

Reply:
kind of 'fail early, fail hard' if you forget about an asset

Reply:
👍 Sounds good. I like this more now come to think of it. Explicit rocks.